### ISSUE TITLE
Translation Request for Stripe Checkout (Tips & Donations): pt-BR

### ISSUE THREAD

**Issue by angelopublio** [TryGhost/Ghost#21766]
### Issue Summary

Hi there!

I wanted to reach out because I've noticed that when I use the “Tips & Donations” feature in pt-BR, the English phrase “Add a personal note” still appears. The other labels are correctly displayed in Brazilian Portuguese, but this one is still in English. It seems to be a custom field generated by Ghost when calling Stripe Checkout.

I thought it might be helpful to suggest a translation: how about “Deixar comentário” for that phrase?

If you need any more information or have questions, I'm here to help!

Best,  
Angelo

![ghost-stripe-checkout-tip-jar-json](https://github.com/user-attachments/assets/e2a01b6a-3982-417c-aac2-3a35c958927d)

![ghost-stripe-checkout-tip-jar](https://github.com/user-attachments/assets/8909c4ec-2c4f-4b89-bee4-f84f7b993cb0)

### Steps to Reproduce

- Set the publication language to "pt-BR" in the General settings.  
- Enable the Tips & Donations feature https://ghost.org/help/tips-and-donations/.  
- Open the shareable link displayed in the Tips & Donations settings.

### Ghost Version

Ghost Pro

### Node.js Version

Ghost Pro

### How did you install Ghost?

Ghost Pro

### Database type

MySQL 5.7

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by cathysarisky** [comment]
Good catch!  I took a quick look, and unfortunately that string is baked into Ghost's Stripe API library.  So it isn't as simple as just wrapping another string - either we punch the 't' helper into the Stripe API library (ugh), or else we pass that string in from Portal.  I'm thinking passing it in from Portal is going to be the best option, since the code is already set up in Portal to translate it.

Adding this to my personal list of translation issues.  Won't be this week.  :)

**Comment by angelopublio** [comment]
Hi Cathy,  
  
Thank you for your prompt response!  
  
Passing the string from the portal seems to be the best approach. I am happy to assist with testing if needed. I appreciate your efforts!  
  
Best,  
Angelo

**Comment by cathysarisky** [comment]
I'm planning to do this, just haven't gotten there yet. :)
